File: Reference/DeltaChallenge-master/DeltaChallenge-master/app/src/main/java/com/example/admin/deltachallenge/data/models/NumbersResponse.java

package com.example.admin.deltachallenge.data.models;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class NumbersResponse {

    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("length")
    @Expose
    private Integer length;
    @SerializedName("data")
    @Expose
    private List<Integer> data = null;
    @SerializedName("success")
    @Expose
    private Boolean success;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Integer getLength() {
        return length;
    }

    public void setLength(Integer length) {
        this.length = length;
    }

    public List<Integer> getData() {
        return data;
    }

    public void setData(List<Integer> data) {
        this.data = data;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

}

File: Week7/Week7Day4Test/app/src/main/java/com/example/luisenriquez/week7day4test/data/remote/model/NumbersResponse.kt
package com.example.luisenriquez.week7day4test.data.remote.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName



class NumbersResponse
{
    @SerializedName("type")
    @Expose
    private var type: String? = null
    @SerializedName("length")
    @Expose
    private var length: Int? = null
    @SerializedName("data")
    @Expose
    private var data: List<Int>? = null
    @SerializedName("success")
    @Expose
    private var success: Boolean? = null

    fun getType(): String? {
        return type
    }

    fun setType(type: String) {
        this.type = type
    }

    fun getLength(): Int? {
        return length
    }

    fun setLength(length: Int?) {
        this.length = length
    }

    fun getData(): List<Int>? {
        return data
    }

    fun setData(data: List<Int>) {
        this.data = data
    }

    fun getSuccess(): Boolean? {
        return success
    }

    fun setSuccess(success: Boolean?) {
        this.success = success
    }

}